lib/assert.js

```diff
@@ -386,6 +387,25 @@ class Assertions {
 			}
 		});
 
+		this.like = withSkip((actual, likePattern, message) => {
+			if (!checkMessage('like', message)) {
+				return;
+			}
+
+			const {actualPicked, result} = likePatternConcordance.compare(actual, likePattern, concordanceOptions);
+			if (result.pass) {
+				pass();
+			} else {
+				const actualDescriptor = result.actual || concordance.describe(actualPicked, concordanceOptions);
+				const expectedDescriptor = result.expected || concordance.describe(likePattern, concordanceOptions);
```

Comment:
When would `actual` and `expected` not exist?

Reply:
I just went with how `deepEqual` [is implemented](https://github.com/avajs/ava/pull/2490/files#diff-f76cd6d4ba3dad8b6b9a393af1fce15bR360). This test errors if `|| ...` part is removed: [`assertions.like('foo', 'bar')`](https://github.com/avajs/ava/pull/2490/files#diff-7ebbd1d7d5643b04618ce7d1c59ce9b8R969)